### PR TITLE
Improve tutorial

### PR DESCRIPTION
**Pull request by natke**

### CODE REVIEW

**Review by sophies927 — COMMENTED** [review]
I think this still puts some of the burden on the user to read through every installation option, whereas there could just be a section at the top with something like this:
```
**Unsure about which installation instructions to follow?** Here's a bit more guidance:

Are you on Windows?

* Yes &rarr; Follow the instructions for DirectML<link to DML instructions>.
* No &rarr; Do you have an NVIDIA GPU?
  * Yes &rarr; Follow the instructions for NVIDIA CUDA GPU <link to CUDA instructions>.
  * No &rarr; Follow the instructions for CPU and Mobile <link to CPU instructions>.
 
*Note: Only one package is required based on your hardware.*
```

This way, people who already know their hardware don't have to read through guidance about how to determine which hardware they have, and people who are unsure about their hardware don't need to read through all of the installation instructions.

**Comment by natke** [comment]
> I think this still puts some of the burden on the user to read through every installation option, whereas there could just be a section at the top with something like this:
> 
> ```
> **Unsure about which installation instructions to follow?** Here's a bit more guidance:
> 
> Are you on Windows?
> 
> * Yes &rarr; Follow the instructions for DirectML<link to DML instructions>.
> * No &rarr; Do you have an NVIDIA GPU?
>   * Yes &rarr; Follow the instructions for NVIDIA CUDA GPU <link to CUDA instructions>.
>   * No &rarr; Follow the instructions for CPU and Mobile <link to CPU instructions>.
>  
> *Note: Only one package is required based on your hardware.*
> ```
> 
> This way, people who already know their hardware don't have to read through guidance about how to determine which hardware they have, and people who are unsure about their hardware don't need to read through all of the installation instructions.

@sophies927 Do you want to edit as per your comment above? Or just add a PR suggestion?

**Comment by sophies927** [comment]
@natke I can go ahead and add a PR suggestion!

File: examples/python/phi-3-tutorial.md [hunk 1]
```diff
@@ -26,25 +26,33 @@ These model repositories have models that run with DirectML, CPU and CUDA.
 
 ## Install the generate() API package
 
-### DirectML
+You only need to install one of these packages, depending on your hardware configuration.
```
Comment (sophies927) [inline examples/python/phi-3-tutorial.md:29]:
```suggestion
**Unsure about which installation instructions to follow?** Here's a bit more guidance:

Are you on Windows machine with GPU?
* I don't know &rarr; Review [this guide](https://www.microsoft.com/en-us/windows/learning-center/how-to-check-gpu) to see whether you have a GPU in your Windows machine.
* Yes &rarr; Follow the instructions for [DirectML](#directml).
* No &rarr; Do you have an NVIDIA GPU?
  * I don't know &rarr; Review [this guide](https://docs.nvidia.com/cuda/cuda-installation-guide-microsoft-windows/index.html#verify-you-have-a-cuda-capable-gpu) to see whether you have a CUDA-capable GPU.
  * Yes &rarr; Follow the instructions for [NVIDIA CUDA GPU](#nvidia-cuda-gpu).
  * No &rarr; Follow the instructions for [CPU](#cpu).
 
*Note: Only one package is required based on your hardware.*
```

File: examples/python/phi-3-tutorial.md [hunk 2]
```diff
@@ -26,25 +26,33 @@ These model repositories have models that run with DirectML, CPU and CUDA.
...
+
+### Option 1: On Windows
```
Comment (sophies927) [inline examples/python/phi-3-tutorial.md:31]:
```suggestion
### DirectML
```

File: examples/python/phi-3-tutorial.md [hunk 3]
```diff
@@ -26,25 +26,33 @@ These model repositories have models that run with DirectML, CPU and CUDA.
...
+
+Install the DirectML package when you are running a Windows machine with a GPU. If you don't know whether you have a GPU in your Windows machine, you can check with this [guide](https://www.microsoft.com/en-us/windows/learning-center/how-to-check-gpu).
 
 ```
 pip install numpy
 pip install --pre onnxruntime-genai-directml
 ```
 
-### CPU
+### Option 2: Other operating systems with CUDA
```
Comment (sophies927) [inline examples/python/phi-3-tutorial.md:40]:
```suggestion
### NVIDIA CUDA GPU
```

File: examples/python/phi-3-tutorial.md [hunk 4]
```diff
@@ -26,25 +26,33 @@ These model repositories have models that run with DirectML, CPU and CUDA.
...
+Install the CUDA package when you have CUDA installed. You can verify whether you have a CUDA capable GPU using this [guide](https://docs.nvidia.com/cuda/cuda-installation-guide-microsoft-windows/index.html#verify-you-have-a-cuda-capable-gpu)
 
 ```
 pip install numpy
-pip install --pre onnxruntime-genai
+pip install --pre onnxruntime-genai-cuda --index-url=https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/onnxruntime-genai/pypi/simple/
 ```
 
-### CUDA
+### Option 3: Other configurations
```
Comment (sophies927) [inline examples/python/phi-3-tutorial.md:49]:
```suggestion
### CPU
```

File: examples/python/phi-3-tutorial.md [hunk 5]
```diff
@@ -26,25 +26,33 @@ These model repositories have models that run with DirectML, CPU and CUDA.
 
 ## Install the generate() API package
 
-### DirectML
+You only need to install one of these packages, depending on your hardware configuration.
+
+### Option 1: On Windows
+
+Install the DirectML package when you are running a Windows machine with a GPU. If you don't know whether you have a GPU in your Windows machine, you can check with this [guide](https://www.microsoft.com/en-us/windows/learning-center/how-to-check-gpu).
```
Comment (sophies927) [inline examples/python/phi-3-tutorial.md:33]:
```suggestion
```

File: examples/python/phi-3-tutorial.md [hunk 6]
```diff
@@ -26,25 +26,33 @@ These model repositories have models that run with DirectML, CPU and CUDA.
...
 
 ```
 pip install numpy
 pip install --pre onnxruntime-genai-directml
 ```
 
-### CPU
+### Option 2: Other operating systems with CUDA
+
+Install the CUDA package when you have CUDA installed. You can verify whether you have a CUDA capable GPU using this [guide](https://docs.nvidia.com/cuda/cuda-installation-guide-microsoft-windows/index.html#verify-you-have-a-cuda-capable-gpu)
```
Comment (sophies927) [inline examples/python/phi-3-tutorial.md:42]:
```suggestion
```

File: examples/python/phi-3-tutorial.md [hunk 7]
```diff
@@ -26,25 +26,33 @@ These model repositories have models that run with DirectML, CPU and CUDA.
...
 ```
 pip install numpy
-pip install --pre onnxruntime-genai
+pip install --pre onnxruntime-genai-cuda --index-url=https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/onnxruntime-genai/pypi/simple/
 ```
 
-### CUDA
+### Option 3: Other configurations
+
+If you are not running on a GPU, use the CPU package.
```
Comment (sophies927) [inline examples/python/phi-3-tutorial.md:51]:
```suggestion
```

**Review by sophies927 — DISMISSED** [review]

**Review by kunal-vaishnavi — APPROVED** [review]

**Pull request merged**